src/python/grpcio_test/grpc_test/framework/face/testing/future_invocation_asynchronous_event_service_test_case.py

```diff
@@ -306,9 +308,10 @@ def testParallelInvocations(self):
         second_request = test_messages.request()
 
         first_response_future = self.stub.future_value_in_value_out(
-            name, first_request, _TIMEOUT)
+            name, first_request, test_constants.SHORT_TIMEOUT)
+        # TODO(bug 2039): use SHORT_TIMEOUT instead
```

Comment:
It's that _TIMEOUT was originally approximately the value of SHORT_TIMEOUT, and we've changed it to LONG_TIMEOUT to avoid flakiness. I presumed you meant for us to eventually change back to SHORT_TIMEOUT after issues underlying the problem (if any addressable) were resolved. Was that not the case?